.github/workflows/trigger_prerelease.yml

```diff
@@ -0,0 +1,46 @@
+name: Trigger prerelease creation
+
+# This workflow triggers a prerelease creation with changelog and the release notes created by the release toolkit.
+# This workflow should be triggered merely from the default branch.
+# If you wish to be 100% free creating a prerelease, just create it manually.
+
+on:
+  workflow_dispatch:
+## TODO here we will need a cronjob.
+
+jobs:
+  prerelease:
+    name: Create prerelease
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.version.outputs.next-version }}
+      image-tag: ${{ steps.image-tag.outputs.tag }}
+    steps:
+      - name: Check is triggered for default branch
+        run: |
+          if [ ${GITHUB_REF_NAME} != ${{ github.event.repository.default_branch }} ] || [ ${GITHUB_REF_TYPE} != "branch" ];then
+            echo This workflow should only be triggered for the '${{ github.event.repository.default_branch }}' branch
+            exit 1
+          fi
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          ref: ${{ github.event.repository.default_branch }}
+          token: "${{ secrets.COREINT_BOT_TOKEN }}"
+      - uses: newrelic/release-toolkit/contrib/ohi-release-notes@v1
+        id: release-data
+      ## TODO here in case the release is empty or on hold we should stop gracefully.
```

Comment:
I would put the ticket of this PR on block until we finish the `is-empty` of the release toolkit and change actions accordingly so we do not have to create a new PR to only change this.

Reply:
It is always better to ship small code changes to simplify reviews/rebases.

Moreover, we will need to add as well the cronjob logic (even if it would be better to have it in a third PR)

Reply:
Fair enough.